frontend/pages/policies/ManagePoliciesPage/components/AddPolicyModal/_styles.scss

```diff
@@ -1,17 +1,22 @@
 .add-policy-modal {
-  height: 80%;
+  height: 90%;
```

Comment:
90% has better readability with long list of policies,
min-height 460px to not cut off the dropdown and needing to scroll on small heights